pandora/models/pandora.py

```diff
@@ -292,7 +294,18 @@ class SearchResultItem(PandoraModel):
 
     @property
     def is_song(self):
-        return self.song_name is not None
+        # Song result tokens start with 'S', followed by seven digits.
+        return re.compile('^([S])(\d{7})$').match(self.token)
```

Comment:
I think ``startswith`` would actually be better as I have just come across a few instances where ``musicToken`` contains more or fewer digits.

Fixed in https://github.com/mcrute/pydora/pull/44/commits/27556ee4085d877e0bf689b742145a3ff46272c6.